commit 8c02c92ab962107ee43c71854bd9712cc492046e
Author: wm4 <wm4@nowhere>
Date:   Fri Apr 15 09:45:15 2016 +0200

vo_opengl: rpi: don't include x11 header file

Copy & paste bug.
---

diff --git a/video/out/opengl/context_rpi.c b/video/out/opengl/context_rpi.c
index c01c173..c0ca733 100644
--- a/video/out/opengl/context_rpi.c
+++ b/video/out/opengl/context_rpi.c
@@ -19,7 +19,6 @@
 #include <assert.h>

 #include "common/common.h"
-#include "video/out/x11_common.h"
 #include "context.h"

 #include "context_rpi.h"
